src/inmanta/module.py

```diff
@@ -510,6 +510,11 @@ def get_namespace_package_name(cls, module_name: str) -> str:
         return f"{const.PLUGINS_PACKAGE}.{module_name}"
 
     def install(self, project: "Project", module_spec: List[InmantaModuleRequirement]) -> Optional["ModuleV2"]:
+        if not self.urls:
+            raise Exception(
```

Comment:
Could we make this more actionable by either referring to the docs or give the exact snippet that fixes this?

Reply:
Added a snippet because referring to the appropriate docs is not straightforward. New error message is:
```
Could not load module std
caused by:
  Exception: Attempting to install a v2 module but no v2 module source is configured. Add at least one repo of type "package" to the project config file. e.g. to add PyPi as a module source, add the following to the `repo` section of the project's `project.yml`:
	- type: package
	  url: https://pypi.org/simple
```

Reply:
@arnaudsjs rerequesting review for this change